SocketHttpListener/Net/HttpListenerResponse.Managed.cs

```diff
@@ -264,7 +260,7 @@ internal void SendHeaders(bool closing, MemoryStream ms, bool isWebSocketHandsha
                 ComputeCookies();
             }
 
-            Encoding encoding = _textEncoding.GetDefaultEncoding();
+            Encoding encoding = Encoding.UTF8;
```

Comment:
Same about uncertainty of utf-8 being the only default.

Reply:
https://github.com/jellyfin/jellyfin/blob/c2dd5baab7d4ff2c5cf089c394e3d7b12dd7de20/SocketHttpListener/Primitives/ITextEncoding.cs